WTWSMS/common/minor_titles/WtWSMS_minor_titles.txt

```diff
@@ -132,32 +132,37 @@ title_sigecempa = {
 }
 
 title_court_celtic_poet = { # Celtic Poet/Bard
-	dignity = 0.25
+	dignity = 1
 	realm_in_name = yes
 	grant_limit = 1
 	opinion_effect = 30
 	
 	monthly_salary = 0.02
-	monthly_prestige = 0.01
+	monthly_prestige = 0.1
```

Comment:
You are wrong and you have no reason to oppose my rectification of your crappy file. It is better, it is an improvement, and it solves the disgusting titles. Your file was utter  crap. Your mercenaries sucked. They were terrible. You have no reason to not implement this change. I did your service.

You are also a monstrous hypocrite: remember when you forced me to change "DONAMONAPRISC_DECADENT" because it "isn't relevant" or connected to the Modifer's purpose?

Yet, "SUNNI_BERBER_COMPANY" for a VALENTIANIAN HOLY ORDER is okay with you? You are a hypocrite!